src/main/java/org/pawpal/model/Pet.java

```diff
@@ -15,97 +25,23 @@ public class Pet {
     @Column
     String breed;
 
-    @Column
-    int age;
+    @Column(name = "date_of_birth")
+    LocalDate dateOfBirth;
 
     @Column
-    int weight;
+    AnimalType type;
 
     @Column
-    boolean gender;
+    double weight;
+
+    @Column(name = "is_male")
+    boolean isMale;
```

Comment:
It would be better to name this variable gender, and establish a convention(0 for male, 1 for female, or opposite) and comment the convention on top of the field

Reply:
If the name would be gender, then we need to create a Enum for all genders because gender is generic. For boolean we go for "is..." because is more specific.If we use boolean gender then it can create a level of confusion.